history.md

```diff
@@ -0,0 +1,81 @@
+RIT LibreCorps has been partnered intermittently with the UNICEF Office of Innovation since 2014.
+As a successful pilot, LibreCorps reached up to 24 supported faculty-driven software projects!
+
+
+
+## FOSS Minor
+
+Beginning of the Fall 2014 Semester, Rochester Institute of Technology started to offer the first minor in [Free and Open Source Software and Free Culture](https://www.rit.edu/study/free-and-open-source-software-and-free-culture-minor).
+The minor was brought about by additional courses offered by the School of Interactive Games and Media ([IGM](https://www.rit.edu/computing/school-interactive-games-and-media)).
+
+
+
+## FOSS@RIT:
+
+FOSS@RIT is the name of the GitHub organization for all things open-source at Rochester Institute of Technology, however, the organization isn't very official. 
+During the period of renaming from FOSSBOX to FOSS@MAGIC, this was what you would call the Open-Source hub, though interchangeable with the FOSS@MAGIC name.
+The FOSS@RIT community was full of people with an interest in free and open-source software at RIT. At the time, FOSS@RIT had the initial purpose of keeping FOSSBOX alive.
+
+
+
+## FOSS@MAGIC:
+
+When [Magic Spell Studio](https://www.rit.edu/magic/magic-spell-studios) was conceptualized, this organization was established to prioritize game development, which was where open-source got started on this campus.
+This organization was managed by [Andrew Phelps](https://professorandrewphelps.net/) in 2015. Since the magic building was not completed at the time, they were relocated to the innovation center, in the exact location that FOSSBOX initially used. 
+FOSS Hours were introduced, which helped increase the involvement of students and other interested parties. However, shortly after, Remy DeCausemaker left Rochester Institute of Technology to go work for Twitter, which made the open-source community lose its growing momentum.
+The organization still had funding and students were still getting involved, events were being run and the organization finally was relocated to inside Magic Spell Studio, which was newly constructed. FOSS@Magic existed in this manner for quite a while, but things changed once Stephen Jacobs got conversations started with UNICEF Innovation, a Tech R&D department permanently part of the United Nations System.
+
+
+
+## OPEN@RIT:
+
+In 2020, the [Alfred P. Sloan Foundation](https://sloan.org/) provided [financial support](https://sloan.org/grant-detail/10076) for an [open work](https://openworkdefinition.com/) organization led by the Rochester Institute of Technology’s Stephen Jacobs: [Open@RIT](https://openr.it/about/). 
+This program was situated just under Rochester Institute of Technologies Vice President for Research, [Ryne Raffaelle](https://www.rit.edu/directory/rprsps-ryne-raffaelle) and has been the home of open-source at RIT for the last four years. 
```

Comment:
```suggestion
This program was situated just under Rochester Institute of Technology Vice President for Research, [Ryne Raffaelle](https://www.rit.edu/directory/rprsps-ryne-raffaelle) and has been the home of Open Source at RIT since 2020.
```

"Past four years" will get outdated quickly, best to use absolute dates instead of relative dates.